scripts/utils.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# To use these utilities you can source this file in your script
```

Comment:
Taken from https://github.com/DataBiosphere/terra-java-project-template/pull/23/files#diff-1615c06ff867567fde8b4caed7973fae157a4e384ebb043edc20cd2dd3a67bb1R2

Reply:
Yes, I think the files that are copied from the java template should have some sort of reference back so folks can update them.

Reply:
`#done` and `#added`